core/unit_test/TestSubView_c15.hpp

```diff
@@ -0,0 +1,28 @@
+//@HEADER
+// ************************************************************************
+//
+//                        Kokkos v. 4.0
+//       Copyright (2022) National Technology & Engineering
+//               Solutions of Sandia, LLC (NTESS).
+//
+// Under the terms of Contract DE-NA0003525 with NTESS,
+// the U.S. Government retains certain rights in this software.
+//
+// Part of Kokkos, under the Apache License v2.0 with LLVM Exceptions.
+// See https://kokkos.org/LICENSE for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//@HEADER
+
+#ifndef KOKKOS_TEST_SUBVIEW_C15_HPP
+#define KOKKOS_TEST_SUBVIEW_C15_HPP
+#include <TestViewSubview.hpp>
```

Comment:
I don't think we need any of the combinatorial machinery in the header file, so you can just include that standalone test (test_subview_constructor_types) directly here or?